1.7/administration/installing/cloud/gce.md

```diff
@@ -22,10 +22,10 @@ A bootstrap node is required to run the scripts and to bootstrap the DC/OS clust
 
 1.  Create the bootstrap node using the Google Cloud Console. In these examples we used a [n1-standard-1](https://cloud.google.com/compute/docs/machine-types) instance running CentOS 7 with a 10 GB persistent disk in `zone europe-west1-c`. The bootstrap node must have "Allow full access to all Cloud APIs" in the Identity and API access section. Also enable Block project-wide SSH keys in the SSH Keys section. Create the instance.
 
-1.  After creating the micro instance, start the instance and run the following from the shell. These commands install prerequisite software on your bootstrap node.
+1.  After creating the boot instance, start the instance and run the following from the shell. These commands install prerequisite software on your bootstrap node.
 
     ```bash
-    $ gcloud components update &&
+    $ sudo gcloud components update &&
```

Comment:
@lingmann, you are right about that. It should say

gcloud components update &&


I have also tested this on a micro instance and it does work with the sudo.
I'm relatively new to using github with a community so if there is anything
I need todo or for some reason I don't respond then please use my name and
I will respond :)

On Wed, 3 Aug 2016 at 23:27 lingmann <notifications@github.com> wrote:

> In 1.7/administration/installing/cloud/gce.md
> <https://github.com/dcos/dcos-docs/pull/476#discussion_r73433163>:
>
> >
> >      ```bash
> > -    $ gcloud components update &&
> > +    $ sudo gcloud components update &&
>
> I believe that best practices are to have gcloud installed with user
> privs, so running the components update as root should not be necessary nor
> desirable, as that would mutate the privileges of the user installed gcloud
> instance.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dcos/dcos-docs/pull/476/files/d64eae18df9c173bdebb857a47e92312f41d7068#r73433163>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABwTc8OQUEIh7tjxYynpQFB7tHsPeKlEks5qcRW2gaJpZM4JY_a8>
> .
>
-- 
Regards

Abdul Jabbar Azam
twitter: @ajazam